Assets/Scripts/Puzzle/CoOpTriggerHandler.cs

```diff
@@ -21,48 +21,29 @@ public class CoOpTriggerHandler : MonoBehaviour
 
         public void GhostInteract()
         {
-            if (!canActivate)
-            {
-                return;
-            }
-            
+            if (!canActivate) return;
+            if (!ghostIsInteracting) return;
             StartCoroutine(TimedDialogue());
             CheckToStartQte();
         }
 
         public void HumanInteract()
         {
-            if (!canActivate)
-            {
-                return;
-            }
-            
+            if (!canActivate) return;
+            if (!humanIsInteracting) return;
             StartCoroutine(TimedDialogue());
             CheckToStartQte();
         }
 
         private void CheckToStartQte()
         {
-            if (!ghostIsInteracting || !humanIsInteracting || !canActivate)
-            {
-                return;
-            }
-
+            if (!ghostIsInteracting || !humanIsInteracting || !canActivate) return;
```

Comment:
Bracers again. This might be part of your IDE's formatting settings but you can change it in Settings (ctrl + alt + S) then Editor>C#>Bracers Layout tab and it will clean up after the settings you set there.